contrib/systemd/gitea.service

```diff
@@ -17,17 +17,16 @@ After=network.target
 #LimitNOFILE=65535
 RestartSec=2s
 Type=simple
-User=git
-Group=git
-WorkingDirectory=/var/lib/gitea/
-ExecStart=/usr/local/bin/gitea web -c /etc/gitea/app.ini
+User=root
```

Comment:
Gitea should not be run as root.

Reply:
Risk averse party poopers...

Reply:
Fine. I'll follow the binary install instructions on the actual Gitea site and see if the service works with the gitea.service file.